lib/util/ws/notify/order_executed.js

```diff
@@ -3,14 +3,16 @@
 const _flatten = require('lodash/flatten')
 const _isFinite = require('lodash/isFinite')
 const _capitalize = require('lodash/capitalize')
+const _lowerCase = require('lodash/lowerCase')
 const notifySuccess = require('./success')
 
 module.exports = (ws, exID, order) => {
-  const { amount, symbol, price, type } = order
+  const { id, amount, symbol, price, type } = order
 
   notifySuccess(ws, _flatten([
-    'Executed', _capitalize(exID), type, amount > 0 ? 'BUY' : 'SELL',
-    'order on', `${symbol}`,
-    _isFinite(+amount) && _isFinite(+price) && +price > 0 && [+amount, '@', price]
+    _capitalize(exID), _lowerCase(type), amount > 0 ? 'BUY' : 'SELL',
+    'order of', _isFinite(+amount) && +amount, symbol,
+    'has been fully executed', isFinite(+price) && +price > 0 && ['at', price],
```

Comment:
if order details is received in `oc` event, it doesn't necessarily mean the order is `fully executed`. Instead, may be we can use the `originalAmount` from `order` to show the amount filled instead of saying fully executed.

Reply:
@avsek477 this function will be called when status is `EXECUTED`. It always means `fully executed`, isn't? https://github.com/vitaliystoliarovcc/bfx-hf-server/blob/ff75a01c593f4cb3d838b3730eec431c099792b3/lib/ws_servers/api/open_auth_bitfinex_connection.js#L72

Reply:
apparently not.. its not the case if the order is partially filled.. when an order is partially filled, it will still send `oc` event with status as `executed` and `originalAmount` as the filled amount. And, it will send an `ou` event as well with the amount updated with status as 'partially filled'